scripts/redact_cli_py/instruction.md

```diff
@@ -0,0 +1,102 @@
+# How to use the Redaction Tool?
+
+## Overview
+The Redaction Tool is provided by Azure Form Recognizer team to help our customers who are willing to share/donate data with Microsoft that can share/donate data efficiently without privacy concern. It is focused on Personally Identifiable Information (PII) or sensitive data labeling and redaction while keeping the semantics of these fields (e.g. length of the values, character/digit patterns, upper/lower case etc.) for prebuilt model training. This is a step-by-step instruction to guide our you how to use the tool.
```

Comment:
nitpick: `It is focused on...` -> `It focuses on...`

Reply:
Thanks! Updated.